subsys/bluetooth/host/gatt.c

```diff
@@ -658,6 +715,23 @@ ssize_t bt_gatt_attr_write_ccc(struct bt_conn *conn,
 	/* Update cfg if don't match */
 	if (ccc->cfg[i].value != ccc->value) {
 		gatt_ccc_changed(attr, ccc);
+
+#if defined(CONFIG_BT_SETTINGS_CCC_STORE_ON_WRITE)
+		if (gatt_ccc_is_conn_queued(conn)) {
+			BT_DBG("Already queued");
+			return BT_GATT_ERR(BT_ATT_ERR_PROCEDURE_IN_PROGRESS);
+		}
+
+		if (bt_addr_le_is_bonded(conn->id, &conn->le.dst)) {
+			/* Store the connection with the same index it has in
+			 * the conns array
+			 */
+			gatt_ccc_store.conn_list[bt_conn_get_id(conn)] =
+				bt_conn_ref(conn);
+			k_delayed_work_submit(&gatt_ccc_store.work,
+				CCC_STORE_DELAY);
```

Comment:
This line doesn't look like it's aligned with the opening parenthesis of the first line.

Reply:
Done